pkg/logs/launchers/file/launcher.go

```diff
@@ -56,6 +57,10 @@ type Launcher struct {
 	scanPeriod             time.Duration
 	flarecontroller        *flareController.FlareController
 	tagger                 tagger.Component
+	filesChan              chan []*tailer.File
+	addedSourceTailers     []*tailer.File
```

Comment:
NIT: Can we choose a better name for this var? `sources` isn't really accurate since it's a slice of files.
I'm thinking something like `filesTailedBetweenScans`

Reply:
renamed 👍